yt/frontends/ramses/data_structures.py

```diff
@@ -656,6 +659,7 @@ def __init__(
         max_level=None,
         max_level_convention=None,
         default_species_fields=None,
+        preserve_symlinks=False,
```

Comment:
If we're going to add more flags here, let's make them keyword only
```suggestion
        *,
        preserve_symlinks=False,
```